test/seam/connect/client-session-token.test.ts

```diff
@@ -92,6 +92,6 @@ test('SeamHttp: updateClientSessionToken checks clientSessionToken is authorized
       instanceOf: SeamHttpApiError,
     },
   )
-  t.is(err?.statusCode, 404)
-  t.is(err?.code, 'client_session_token_not_found')
+  t.is(err?.statusCode, 401)
+  t.is(err?.code, 'unauthorized')
```

Comment:
Changed the assertions according to [this](https://hello-seam.slack.com/archives/C07DTP6EGBZ/p1731963972774479?thread_ts=1731528462.613209&cid=C07DTP6EGBZ)